### DISCUSSION TITLE
JavaScript Primerを読んだ感想を貼る場所

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1305] [体験のシェア]
JavaScript Primerを読んだ感想や感想を書いたブログやTweetなどを貼るスレッドです。

単純に読んだ感想を書くだけでもいいし、リンクを貼るだけでもいいと思います！

書籍版の感想はAmazonのレビューに書くのもいいと思います。

- [JavaScript Primer 迷わないための入門書 | azu, Suguru Inatomi |本 | 通販 | Amazon](https://www.amazon.co.jp/dp/4048930737/)

**Comment by Akira6I6I** [comment]
早速トライします
